### ISSUE TITLE
NTR: Borrelia burgdorferi antibody high in CSF, blood or synovium (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8737]
New term request Borrelia burgdorferi antibody high in CSF, blood or synovium (samant_101219003110): Lyme bacterium; cerebrospinal fluid; IgM or IgG

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000927
- label: Positive CSF Borrelia burgdorferi antibody positivity
- parents: [Positive CSF pathogen-specific antibody positivity (HP:0430139)](https://hpo.jax.org/app/browse/term/HP:0430139)
- pmid: 
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of Borrelia burgdorferi.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8737
- synonyms:
